pkg/network/usm/shared_libraries.go

```diff
@@ -197,24 +197,43 @@ func (w *soWatcher) Start() {
 		log.Errorf("can't initialize process monitor %s", err)
 		return
 	}
-	cleanupExit, err := w.processMonitor.Subscribe(&monitor.ProcessCallback{
-		Event:    monitor.EXIT,
-		Metadata: monitor.ANY,
-		Callback: w.registry.unregister,
+
+	cleanupExit, err := w.processMonitor.SubscribeExit(&monitor.ProcessCallback{
+		FilterType: monitor.ANY,
+		Callback:   w.registry.unregister,
 	})
 	if err != nil {
 		log.Errorf("can't subscribe to process monitor exit event %s", err)
 		return
 	}
 
 	go func() {
+		processSync := time.NewTicker(time.Minute)
```

Comment:
shall we configure this, host with tons of containers, would require adjustment probably